src/scss/_diver.scss

```diff
@@ -8,9 +8,21 @@
 
   padding-top: 460px;
 
+  @media screen and (max-width: $xxl) {
+    padding-top: 350px;
+    background: url('../images/diver_bubble.png') 100% no-repeat,
```

Comment:
try to change only `background-image`

Reply:
then there will be no gradient on the background
![q1](https://user-images.githubusercontent.com/62384781/183734630-6ced75e3-117b-484b-bc08-1c3418909bad.png)